terraform/aws/national-parks.tf

```diff
@@ -211,13 +213,14 @@ resource "aws_instance" "national_parks" {
       "chmod +x /tmp/install_hab.sh",
       "sudo /tmp/install_hab.sh",
       "sudo hab license accept",
+      "sudo hab pkg install core/hab-sup/0.83.0-dev -c unstable",
       "sudo mv /home/${var.aws_ami_user}/hab-sup.service /etc/systemd/system/hab-sup.service",
       "sudo systemctl daemon-reload",
       "sudo systemctl start hab-sup",
       "sudo systemctl enable hab-sup",
       "sleep ${var.sleep}",
       "sudo mkdir -p /hab/user/haproxy/config /hab/user/config-baseline/config /hab/user/audit-baseline/config",
-      "sudo chown hab:hab -p /hab/user",
+      "sudo chown hab:hab /hab/user",
```

Comment:
chown -R ?

Reply:
I wasn't sure about this one either, It was changed in your branch I started from, that's where I pulled it.

Reply:
Hehe, I just got annoyed that it gave me a (non-breaking) error since chown doesn't have a -p flag. -R does the same thing (directory recursion), so I figure that's what @smford22 meant when he laid down the initial code. It works either way, so it's not crucial.

Reply:
Do you think all the systems should have 

"sudo chown hab:hab -R /hab/user", 

Each instance we load has the sudo chown hab:hab, the perm peer still has -p with the national parks, mongo, haproxy all being just hab:hab with no -p or -R flag